lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_handlers.cpp

```diff
@@ -4406,7 +4406,7 @@ status_code_e s1ap_handle_enb_initiated_reset_ack(
   memset(&pdu, 0, sizeof(pdu));
   pdu.present = S1ap_S1AP_PDU_PR_successfulOutcome;
   pdu.choice.successfulOutcome.procedureCode = S1ap_ProcedureCode_id_Reset;
-  pdu.choice.successfulOutcome.criticality = S1ap_Criticality_ignore;
+  pdu.choice.successfulOutcome.criticality = S1ap_Criticality_reject;
```

Comment:
LGTM, I assume this refers to the "Message Type" row in 9.1.8.2 in the linked document.

https://www.etsi.org/deliver/etsi_ts/136400_136499/136413/15.03.00_60/ts_136413v150300p.pdf

Reply:
Correct, it is reference to 9.1.8.2 where the column "Criticality" is telling us that the criticality is applied (therefore should be included) and the column "Assigned Criticality" is telling us that the value expected is "reject".